packages/lesswrong/components/posts/PostsItemTooltipWrapper.tsx

```diff
@@ -18,11 +18,6 @@ const PostsItemTooltipWrapper = ({children, post, className}: {
         anchorEl={anchorEl}
         clickable={false}
         placement="bottom-end"
-        modifiers={{
-          flip: {
-            enabled: false
-          }
-        }}
```

Comment:
I removed this because I was getting a UI bug. (See here for visual: https://drive.google.com/file/d/1-ly-ycH0AguQqO4NETK3Ww8WWg9J-4au/view?usp=sharing) This can also show up on LessWrong, e.g. [here](https://www.lesswrong.com/groups/emiR5SwosnAdasGeS), by scrolling to the bottom of the page and hovering over the lowest post. If you scroll down while that tooltip is open, you get this.

Separate from that bug, prior to this change posts toward the bottom of the viewport would have tooltips that were mostly offscreen. This has been flagged in our QA checklist for a while: 
![image](https://user-images.githubusercontent.com/25752873/124336075-81b6b200-db51-11eb-8b9d-4ec55389ef93.png)

I do want to avoid Chesterton's fence here though, and I'm not yet sure why this override of MUI's default popper functionality was put in place. @jimrandomh, do you by chance know more?